src/GraphRequestUtil.ts

```diff
@@ -58,3 +58,99 @@ export const serializeContent = (content: any): any => {
 	}
 	return content;
 };
+
+export class PathUtils {
+	private static hostRe = /([a-z]+.microsoft.[a-z]+)/g;
+	private static versionRe = /(beta|v1.0)/g;
```

Comment:
in `(beta|v1.0)` expression, `.` is a wildcard character, so it will match things like: `v1a0` and `v1b0` etc. The expression should be `(beta|v1\.0)`.